src/control/Control.Layers.js

```diff
@@ -198,6 +198,8 @@ export var Layers = Control.extend({
 		var link = this._layersLink = DomUtil.create('a', className + '-toggle', container);
 		link.href = '#';
 		link.title = 'Layers';
+		link.setAttribute('aria-label', 'toggle layer control');

```

Comment:
@rowanwins revisiting this, I think the naming of `aria-label` is a bit verbose. In comparison, zoom controls don't say "Zoom in/out control" or "Press to zoom in/out", they're just "Zoom in/out" (that they're "pressable" or "controls" is implied by the fact that they're buttons, well, `role=button`).

So IMO "Layers" should be perfectly fine (same as `title`). It'd also be more consistent with other controls, since almost all of Leaflet's controls duplicate the name in both `aria-label` and `title`.